{"name": "test_login_with_invalid_email_and_valid_password", "status": "passed", "start": 1734967958199, "stop": 1734967976281, "uuid": "41b04cb7-780f-4c9b-9c40-edb6d36a7138", "historyId": "cd3500ccb6b16a398d3c5c6e5203a23d", "testCaseId": "cd3500ccb6b16a398d3c5c6e5203a23d", "fullName": "tests.test_login.TestLogin#test_login_with_invalid_email_and_valid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "EPINHYDW04A4"}, {"name": "thread", "value": "2156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}